crates/sui-genesis-builder/src/lib.rs

```diff
@@ -1075,6 +1081,24 @@ pub fn generate_genesis_system_object(
             )?;
         }
 
+        if protocol_config.bridge() {
```

Comment:
minor nit: consistency on whitespace -- it seems like all the other protocol-config gated object initialisation blocks got smooshed together (no blank lines between them). I tend to prefer having the blank line, but if we add it here, can we do it consistently?